packages/rancher-monitoring/generated-changes/patch/templates/exporters/node-exporter/servicemonitor.yaml.patch

```diff
@@ -0,0 +1,11 @@
+--- charts-original/templates/exporters/node-exporter/servicemonitor.yaml
++++ charts/templates/exporters/node-exporter/servicemonitor.yaml
+@@ -1,4 +1,7 @@
+-{{- if .Values.nodeExporter.enabled }}
++{{- if (and (not .Values.nodeExporter.enabled) .Values.hardenedNodeExporter.enabled) }}
++{{ required "Cannot set .Values.hardenedNodeExporter.enabled=true when .Values.nodeExporter.enabled=false" "" }}
++{{- end }}
++{{- if (and .Values.nodeExporter.enabled (not .Values.hardenedNodeExporter.enabled)) }}
```

Comment:
hardenedNodeExporter is treated as an optional add-on to nodeExporter; you cannot enable hardenedNodeExporter unless nodeExporter is enabled.

If hardenedNodeExporter is enabled, it disables the default ServiceMonitor that is created for nodeExporter monitoring in favor of one that routes through the proxy.